explorer/explorermiddleware.go

```diff
@@ -7,6 +7,7 @@ import (
 	"context"
 	"net/http"
 	"strconv"
+        "fmt"
```

Comment:
All .go files need to be processed with `go fmt`.  I suggest configuring your editor to do this automatically on save.

I use Visual Studio Code with the Go extension.

Reply:
Also, a good go extension will sort the imports, but leave them in separate blocks like we have done for std lib imports.